client/src/components/Allocations/AllocationsLayout.jsx

```diff
@@ -68,7 +68,7 @@ export const Layout = ({ page }) => {
   return (
     <Section
       bodyClassName="has-loaded-allocations"
-      introMessageName="ALLOCATIONS"
+      messageName="ALLOCATIONS"
```

Comment:
This propagates down to the `IntroMessage` component, so this should probably stay as `introMessageName`

Reply:
@rstijerina Would this be true for `SectionMessages` too?
I wasn't sure what you meant by refactoring references to `introMessageName` in https://github.com/TACC/Core-Portal/pull/615#discussion_r870674573.

Reply:
@duckonomy Ah, sorry, that wasn't clear. I meant to say change references to `introMessageName` if that didn't pertain to the component it was in